src/Model/Booking.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+namespace App\Model;
+
+use DateTime;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class Booking
+{
+
+    /**
+    * @Assert\Length(
+     *      min = 2,
+     *      max = 50,
+     *      minMessage = "votre prénom doit avoir plus de 2 caractères",
+     *      maxMessage = "votre prénom doit avoir moins de 50 caractères",
+     * )
+     * @Assert\Type("string")
+     * @Assert\NotBlank
+     *
+     */
+    private $firstname;
+
+    /**
+     * @Assert\Length(
+     *      min = 2,
+     *      minMessage = "votre Nome doit avoir plus de 2 caractères")
+     * @Assert\Type("string")
+     * @Assert\NotBlank
+     */
+    private $lastname;
+
+
+    /**
+     * @Assert\Length(
+     *      min = 2,
+     *      minMessage = "votre adresse doit avoir plus de 2 caractères")
+     * @Assert\Type("string")
+     * @Assert\NotBlank
+     */
+    private $address;
+
+
+    /**
+     * @Assert\Type("string")
+     * @Assert\NotBlank
+     */
+    private $email;
```

Comment:
There is an email constraint : https://symfony.com/doc/current/reference/constraints.html#string-constraints